x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-config-topics.adoc

```diff
@@ -44,7 +44,7 @@ NOTE: To use the Functional Acknowledgment schema type (`997` or `999`) it must
 |XML Value |Studio Option
 |formValidation="X12"/"HIPAA_SNIP1"/"HIPAA_SNIP2" |The schema form and validation level can be either X12 or HIPAA. The default is X12.
 |schemas=list of values |List of schema paths the connector can use. These paths can be either a file system or `classpath`.
-|characterEncoding="PLATFORM"/"ASCII"/"ISO8859_1"/"IBM1047"/"UTF8" |The character encoding used for both send and receive messages. The default `PLATFORM` is the Mule platform default encoding
+|characterEncoding="PLATFORM"/"ASCII"/"ISO8859_1"/"IBM1047"/"UTF8" |The character encoding used for both send and receive messages. The default `PLATFORM` is the Mule platform default encoding.
```

Comment:
What is "the Mule platform"? We use the abbreviation "Mule" to represent Mule runtime engine.